### ISSUE TITLE
Not working on Xcode 6.3.1

### ISSUE THREAD

**Issue by rrecio** [neonichu/BBUUtilitiesTuckAway#5]
Nothing happens when I start typing

Using OS 10.10.3

**Comment by neonichu** [comment]
Yep, this comment <https://github.com/neonichu/BBUDebuggerTuckAway/issues/17#issuecomment-100455269> also applies here - would love a PR if you have time :)
